company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise4Test.java

```diff
@@ -108,9 +112,8 @@ public void filterOrderValues()
     @Test
     public void filterOrderValuesUsingPrimitives()
     {
-        List<Order> orders = this.company.getMostRecentCustomer().getOrders();
-        MutableDoubleList orderValues = null;
-        MutableDoubleList filtered = null;
+        MutableDoubleList filtered = ListIterate.collectDouble(
```

Comment:
Please break this up into two statements with both `orderValues` and `filtered` variables as the test has.